### ISSUE TITLE
[Bug]: Eccentric Tome loses all books except the active one

### ISSUE THREAD

**Issue by macks2008** [FTBTeam/FTB-Modpack-Issues#2344]
### Modpack

FTB Presents Direwolf20 1.18

### Modpack version

1.10.3

### Log Files

Note that I do not reproduce the bug during this session, I only start the client so you can get all of the juicy version info:

[debug.log](https://github.com/FTBTeam/FTB-Modpack-Issues/files/10250549/debug.log)
[latest.log](https://github.com/FTBTeam/FTB-Modpack-Issues/files/10250552/latest.log)
[console.log](https://github.com/FTBTeam/FTB-Modpack-Issues/files/10250555/console.log)

### Describe the bug

rarely, while using the Eccentric Tome I spawned with (or, technically speaking, a copy of it since this bug keeps breaking it), I'll try to switch books only to discover that the only book available is the one I was just reading, even though I know I never had any Eccentric Tomes except for the one that had "everything" in the pack. In other words, there appears to be a bug with the one of the mods involved causing it to lose the list of inactive books.

### Steps to reproduce

1. Get the Eccentric Tome that you spawn with (or a copy of it. I made a few copies using control-pick-block in creative)
2. right-click with the book in your hand
3. pick a book, any book
4. I'm not entirely sure what I'm doing at this point to trigger the issue. One time it happened when I looked up the "uses" of the PNC Repressurized manual... today it happened (or at least I first noticed it) when I had an Eccentric instance of LaserIO's guide in my offhand. 
5. Hover your cursor over the tome. It might show only one book is "inside" it
6. Left click to revert to the Eccentric Tome
7. Bug occurs. If you left click with the tome now, you'll only see the book that you picked in the first step. 
8. This can NOT be fixed by logging out and back in or other interactions which force the server to resend the book data to you. It appears the data was actually lost server side.

### Expected behaviour

you should still see all the books from before, or at worst it should be client-side so that you can reset it by dropping the book and picking it back up or something.

### Additional information

In the process of trying to figure out how to reproduce this, I noticed that if I inspect one of the  activated guidebooks with KubeJS (even when this bug doesn't happen), it tells me I'm holding a Patchouli book instance, not Eccentric Tome. How does it normally change back from a completely different item class? My current hypothesis is that this apparent item identity change is part of what causes the bug, though I'm struggling to reproduce the problem (predictably, at least) even with that information.

**Comment by macks2008** [comment]
whoops, this is 1.18, I chose the wrong pack

**Comment by macks2008** [comment]
thanks.
By the way this hasn't happened in a while for me, but I don't think I changed anything (like mod or pack versions) and wouldn't rule it out just yet.
When I first created the world it was happening all the time, and gradually got less frequent. I'm wondering if it's some sort of initialization issue with patchouli/patchouli books that only happens one time per book, like when you first open it? I don't see why that would cause it to break the Eccentric Tome (other than obviously the fact it is not actually an eccentric tome at the moment that you open it, it's a patchouli book with extra data), but anything's possible?
